nemo/collections/nlp/modules/common/megatron/enums.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Transformer."""
+import enum
+
+
+class LayerType(enum.Enum):
```

Comment:
Is this copied from apex? If so, you should do a try catch after these definitions, import the megatron / apex equivalents, and assert that the values in apex map exactly to the values inside this layer. 

This can be a unittest inside NeMo so that updates to apex do not fail to be updated inside NeMo.